src/js/components/modals/InstallPackageModal.js

```diff
@@ -89,19 +92,29 @@ class InstallPackageModal
         currentTab: "defaultInstall",
         truncatedPreInstallNotes: true
       });
+
+      return;
     }
-    // If Opening
-    if (!props.open && nextProps.open) {
-      // If default - auto-install
-      if (!nextProps.advancedConfig) {
-        this.handleInstallPackage();
-      } else {
-        // Show advanced install
-        this.setState({
-          currentTab: "advancedInstall"
-        });
-      }
+
+    if (nextProps.isBetaPackage) {
```

Comment:
This could potentially cause a double render if a different prop of the component changed and `nextProps.isBetaPackage` was true and is true with updated props. Can you do something to prevent this?